lib/features/account/report_issue.dart

```diff
@@ -222,6 +236,33 @@ class _ReportIssueState extends State<ReportIssue> {
     }
   }
 
+  Future<void> onShareLogsTap() async {
+    try {
+      AppLoadingDialog.showLoadingDialog(context);
+      await sessionModel.shareLogs();
+      AppLoadingDialog.dismissLoadingDialog(context);
+      CDialog.showInfo(
+        context,
+        title: 'report_sent'.i18n,
+        description: 'thank_you_for_reporting_your_issue'.i18n,
+        actionLabel: 'continue'.i18n,
+        agreeAction: () async {
+          resetController();
+          return true;
+        },
+      );
```

Comment:
Oh, I did notice one thing with the confirmation message. Since I'm not reporting an issue, you could skip this extra dialog entirely or update the text to say logs were successfully shared?

![image](https://github.com/user-attachments/assets/caf9c8b5-7a18-493a-841c-6cec8a06f4b1)